druid/ingestion-spec/ml_project_index.json

```diff
@@ -0,0 +1,285 @@
+{
+  "type": "index",
+  "spec": {
+    "ioConfig": {
+      "type": "index",
+      "inputSource": {
+        "type": "azure",
+        "uris": [
+          "azure://<container_name>/projects/sl_projects.json"
```

Comment:
Same for azure path in this spec

Reply:
can you please share me the sample ingestion spec which is correct ? , because i go throughed the already existing ingestion spec of diksha.

Reply:
Resolved, Please check